include/dem/particle_wall_contact_force.h

```diff
@@ -80,6 +82,56 @@ class ParticleWallContactForce
     return torque_on_walls;
   }
 
+  /**
+   * Carries out the calculation of the contact force for IB particles. This
+   * function is used in fem-dem/ib_particles_dem.
+   *
+   * @param contact_info Contact history including tangential overlap and relative
+   * velocity.
+   * @param normal_force Contact normal force.
+   * @param tangential_force Contact tangential force.
+   * @param tangential_torque
+   * @param rolling_resistance_torque Contact rolling resistance torque.
+   * @param particle
+   * @param wall_youngs_modulus
+   * @param wall_poisson_ratio
+   * @param wall_restitution_coefficient
+   * @param wall_friction_coefficient
+   * @param wall_rolling_friction_coefficient
+   * @param dt Time-step.
```

Comment:
update the parameters list. I think it should also be updated in other modified functions (particle-particle, etc.)